enhancements/storage/csi-certification.md

```diff
@@ -0,0 +1,125 @@
+---
+title: csi-certification
+authors:
+  - "@fbertina"
+reviewers:
+  - "jsafrane"
+  - "@gnufied”
+  - "@chuffman"
+approvers:
+  - "@..."
+creation-date: 2019-12-18
+last-updated: 2019-12-18
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# CSI Certification
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+We want to have a CSI certification suite that our storage vendors can use to certify their CSI drivers.
+
+## Motivation
+
+As storage vendors want to include their solutions into the OpenShift ecosystem, they will need to make their CSI drivers installable via an operator. To get support for their drivers, they need to certify them and we need to provide a way to test that the driver they provide meets the key requirements.
+
+### Goals
+
+The goal is to allow a storage vendor to run a test that validates their CSI drivers for use with OpenShift as a step in the process to certify their operators for OperatorHub.io.
```

Comment:
Note that certified OpenShift operators are listed in OpenShift's own (embedded) OperatorHub, whereas OperatorHub.io is used for operator efforts from the Kubernetes community.

Reply:
Renamed to OperatorHub